src/current/_includes/releases/v24.1/v24.1.0-alpha.1.md

```diff
@@ -140,106 +138,57 @@ Release Date: March 7, 2024
+- Fixed a bug where casts of floats to integers simply truncated the decimal portion. These casts now match the PostgreSQL behavior of rounding to the nearest integer, and in cases of a value halfway between two integers, rounding to the nearest even number. This aligns with the "round half to even" rule or "bankers' rounding", offering greater overall precision across a group of such cast operations. [#117798][#117798]
+- Fixed a bug where statements like `ADD COLUMN j INT, ADD UNIQUE WITHOUT INDEX (j)`, which add new columns with unique constraints without creating associated indexes, would fail with an internal error. [#118291][#118291]
+- Previously, altering from a [Regional By Row]({% link v23.2/alter-table.md}#regional-by-row) table to a Regional By Table table could cause leaseholders to never move to the databse's primary region. This is now fixed. [#118001][#118001]
+- Users with the [VIEWACTIVITY]({% link v23.2/security-reference/authorization.md}#supported-privileges) privilege can now request statement bundles using `crdb_internal.requets_statement_bundle` or through the DB Console [SQL Activity]({% link v23.2/security-reference/authorization.md}#supported-privileges) page. [#118760][#118760]
+- Fixed an internal error with message like: `LeafTxn ... incompatible with locking request` that occurs when performing an update under [`READ COMMITTED` isolation]({% link v23.2/read-committed.md}) which cascades to a table with multiple other foreign keys. [#118722][#118722]
+- Fixed a bug where [ALTER PRIMARY KEY]({% link v23.2/alter-table.md}#alter-primary-key) could fail with an error `non-nullable column <x> with no value! Index scanned ..` when validating recreated secondary indexes. [#118843][#118843]
+- Fixed a bug where a sequence name allocated by `SERIAL` that conflicted with an existing type name would cause an error. [#118861][#118861]
+- Fixed a bug where [`COMMENT ON`]({% link v23.2/comment-on.md}) statements could fail with an "unexpected value" error if multiple COMMENT statements were running concurrently. [#119007][#119007]
+- Fixed a bug where the internal error `attempting to append refresh spans after the tracked timestamp has moved forward` could result in certain cases when using virtual tables, such as `crdb_internal.system_jobs`. The bug was introduced in CockroachDB v23.1. [#119176][#119176]
+- Fixed a bug where operations on the `crdb_internal.leases` table could cause a node to become unavailable due to a deadlock in the leasing subsystem. [#119305][#119305]
+- If an individual replica's circuit breaker had tripped but the range was otherwise functional, for example, because the replica was partially partitioned away from the leaseholder, it was possible for a gateway to persistently error when contacting this replica instead of retrying against a functional leaseholder elsewhere. The gateway will now retry such errors against other replicas once. [#118737][#118737]
```

Comment:
Suggest linking "replica", "leaseholder", and "gateway" to appropriate Architecture section docs.

```
Previously, if an individual replica's circuit breaker had tripped but the range was otherwise functional— for example, because the replica was partially partitioned away from the leaseholder—it was possible for a gateway to persistently error when contacting this replica instead of retrying against a functional leaseholder elsewhere. The gateway will now retry such errors against other replicas once.
```